src/app/pages/Splash/SplashCTA/index.tsx

```diff
@@ -5,7 +5,9 @@ import {
   CTAHeader,
   CTAProductCard,
 } from './styles';
-
+import pen from 'tezospepe/tezos-pepe-frontend/src/app/pages/Splash/SplashCTA/assets/pen_56.png';
```

Comment:
the path provided doesn't exist in the context of the app, you'll need to use relative imports (e.g. ./assets/pen_56.png)